Makefile

```diff
@@ -0,0 +1,65 @@
+DOCKER_COMPOSE ?= docker compose
+FILE ?= local.yml
+DC = $(DOCKER_COMPOSE) --file=$(FILE)
+RUN = $(DC) run --rm
+DJANGO = django
+MANAGE = $(RUN) $(DJANGO) python manage.py
+
+shell:
+	$(MANAGE) shell
+
+
+logs:
+	$(DC) logs -f $(DJANGO)
+
+
+down:
+	$(DC) down
+
+
+up:
+	$(DC) up -d
+
+
+restart: down up
+
+
+build:
+	$(DC) build
+
+
+createsuperuser:
+	$(MANAGE) createsuperuser
+
+
+freshstart: down build up
+
+
+makemigrations:
+	$(MANAGE) makemigrations
+
+
+migrate:
+	$(MANAGE) migrate
+
+
+types:
+	$(RUN) $(DJANGO) mypy pra_request_tracker
+
+
+test:
+	$(RUN) $(DJANGO) coverage run -m pytest
```

Comment:
There's a plugin called `pytest-cov` that lets you do `pytest --cov` instead of running via coverage!
https://pypi.org/project/pytest-cov/

Reply:
Oh sick! Do you mind if I do that in a separate PR?

Reply:
Totally, and I'm also not saying it has to be used here either! Your discretion, I just find it makes things a little easier :smile: